src/main/java/org/jitsi/videobridge/rest/LongPollingServlet.java

```diff
@@ -38,7 +59,55 @@ protected void doGet(
         throws IOException,
                ServletException
     {
-        // TODO Auto-generated method stub
-        response.setStatus(HttpServletResponse.SC_NOT_FOUND);
+        System.out.println("BB: IN LONG POLLING");
+        Continuation continuation = ContinuationSupport.getContinuation(request);
+        
+        // We don't get the target here (TODO: is that something that could
+        //  be fixed?), so we need to do a bit more work to extract the bits of the path
+        //  we're interested in.
+        // The stuff we're interested in starts after the /colibri/
+        String colibriPath = request.getRequestURL().toString().split("colibri/")[1];
+        String target = colibriPath.split("/")[0];
+
+        continuation.suspend();
```

Comment:
Once you've identified a target/HTTP request for asynchronous processing and started it i.e. suspended the request, you are to register the continuation (either Jetty Continuations or Servlet 3.0 API) with an asynchronous service which is to resume the continuation when the waited-for event occurs. In other words, the code bellow should return the control to the calling method as soon as possible and without blocking.